### DISCUSSION TITLE
aplayer 配置

### DISCUSSION THREAD

**Discussion by ly20000814bx** [volantis-x/hexo-theme-volantis#642] [General]
关于 aplayer 那块配置能不能加进主页配置那一栏

**Comment by BeaCox** [comment]
可以描述得清楚一些嘛

**Reply by wndbac** [reply]
他说的是导航栏吗
MHuiG倒是实现了
![image](https://user-images.githubusercontent.com/97789192/213728311-db9af4a7-2121-4325-b889-710b4f94f810.png)

**Reply by BeaCox** [reply]
導航欄音樂播放器是可配置的，不需要額外實現
